routes/cube_routes.js

```diff
@@ -2482,14 +2482,22 @@ router.post('/submitdeck/:id', async (req, res) => {
     const userq = User.findById(deck.owner);
     const cubeOwnerq = User.findById(cube.owner);
 
-    const [user, cubeOwner] = await Promise.all([userq, cubeOwnerq]);
+    var [user, cubeOwner] = await Promise.all([userq, cubeOwnerq]);
```

Comment:
There isn't a strong reason to use `var` over `let` in es6 as a heads up. `let` does allow for reassignment.